cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -159,6 +164,15 @@ function update_addons() {
   fi
 }
 
+function ensure_addons() {
+  # Create objects already exist should fail.
+  # Filter out `AlreadyExists` message to not noisily log.
+  ${KUBECTL} ${KUBECTL_OPTS} create --namespace=${SYSTEM_NAMESPACE} -f ${ADDON_PATH} -l ${SERVICE_ENSURE_EXIST_LABEL} \
```

Comment:
should we just patch kubectl to have a `--ignore-already-exists` flag to create, similar to `--ignore-not-found` on delete?